source/backend/discovery/src/lib/sdkResources/index.js

```diff
@@ -94,13 +94,15 @@ async function getAllSdkResources(accountsMap, awsClient, resources) {
     logger.error(`There were ${firstErrors.length} errors when adding first order SDK resources.`);
     logger.debug('Errors: ', {firstErrors});
 
-    firstResults.flat().forEach(resource => resourcesCopy.push(resource));
+    firstResults.flat().forEach(resource => { if(resource != undefined) { resourcesCopy.push(resource) } });
 
     const secondOrderResourceTypes = new Set(R.keys(secondOrderHandlers));
 
+    let firstResultsFiltered = firstResults.filter(resource => resource != undefined)
```

Comment:
`!= null` will catch both null and undefined

Reply:
See above: we can remove this filtering by returning an empty array in the DynamoDB handler when `configuration.latestStreamArn` does not exist.